spec/lib/attribute_cleaner_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+RSpec.describe AttributeCleaner, type: :lib do
+  with_model :TestClass do
```

Comment:
Reading notes: remember this, I haven't previously seen it and it's super interesting.

Reply:
Once I needed something like this in one of my tests and I had already started implementing it but when it got time to wire up the schema DSL I luckily decided to take a break and look for existing solutions: https://github.com/Casecommons/with_model